search/shortest.go

```diff
@@ -0,0 +1,263 @@
+// Copyright ©2014 The gonum Authors. All rights reserved.
```

Comment:
Actually, I think you are correct for this date. I need to go through these carefully and see which ones are derivative.